alpha-build-machinery/README.md

```diff
@@ -0,0 +1,37 @@
+# library-go/alpha-build-machinery
+These are the building blocks for this and many of our other repositories to share code for Makefiles, helper scripts and other build related machinery.
+
+## Makefiles
+`make/` directory contains several predefined makefiles (*.mk) to choose from and include one of them as a base in your final `Makefile`. These are the predefined flows providing you with e.g. `build`, `test` or `verify` targets. To start with it is recommended you base Makefile on the corresponding `*.example.mk` using copy&paste.
```

Comment:
something is wrong with formatting after this paragraph.

Reply:
the rendered file was ok, but the code highlighting in this PR was off because of unescaped `(*.mk)` - fixed now